### PR TITLE
Was added form

### PR DESCRIPTION
**Pull request by kmp93**
- [DEMO LINK](https://kmp93.github.io/layout_html-form/)
- [TEST REPORT LINK](https://kmp93.github.io/layout_html-form/report/html_report/)

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -13,5 +13,154 @@
...
+            value="12"
+            min="1"
+            max="100"
+          >
+        </label>
+
+        <label class="form-field">
+            Full date of birth:
+          <input 
+            type="text"
```
Comment (valerii-derkach) [inline src/index.html:50]:
```suggestion
            type="date"
```

File: src/index.html [hunk 2]
```diff
@@ -13,5 +13,154 @@
   <body>
     <h1>HTML Form</h1>
     <script type="text/javascript" src="./main.js"></script>
+    <form action="http://localhost:8080/" method="post">
```
Comment (valerii-derkach) [inline src/index.html:16]:
```suggestion
    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
    
```
Please, read the task
On form submit send form data to https://mate-academy-form-lesson.herokuapp.com/create-application

**Review by valerii-derkach — CHANGES_REQUESTED** [review]
Your demo link doesn't work
and on submit there is an error

**Comment by kmp93** [comment]
New

**Pull request closed without merging**